### PR TITLE
Web home layout

### PR DESCRIPTION
**Pull request by dsernahiguita**
Hi

Pull request for the web home layout https://xd.adobe.com/view/cc668fc8-7a1c-47ad-67b8-fa74c58510d8-ba57/screen/0606be88-caaa-435f-8e54-dabfb7becbce/Web-Home

To check this page please use the link localhost:1337/webHome

Thanks in advance
Diana

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -30,6 +31,8 @@ function App() {
   return (
     <ThemeProvider theme={paisanosTheme}>
       <main>
+
+        <Route exact path='/webHome' render={(props) => <WebHome {...props} />} />
         <Route exact path='/home' render={() => <Home />} />
```
Comment (JuanSeBestia) [inline src/App.js:36]: Also, I think we can replace actually home component, what do you think?

File: src/components/LoginRegister/index.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+  render() {
+    return (
+      <div className="webapp-login">
+        <form noValidate autoComplete="off">
+          <Grid container spacing={6}>
+            <Grid item xs={6}>
+              {this.viewFieldsLogin()}
+            </Grid>
+            <Grid item xs={6}>
+              {this.viewFieldsRegister()}
```
Comment (JuanSeBestia) [inline src/components/LoginRegister/index.js:147]: Please use jsx syntax, to do that you need change name function o PascalCase and using like
`<this.ViewFieldsRegister /> `

File: src/translations/es.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "app.title": "COVID-19",
   "app.description": "Deep Vision",
-  "app.welcome": "Bienvenidos a COBIC-19 Diagnosis",
+  "app.welcome": "Bienvenidos a COVIC-19 Diagnosis",
```
Comment (JuanSeBestia) [inline src/translations/es.json:4]: COVID *

File: src/App.js (hunk 1 above)
Comment (dsernahiguita) [reply src/App.js]: done, I did already the changes :)

File: src/components/LoginRegister/index.js (hunk 2 above)
Comment (dsernahiguita) [reply src/components/LoginRegister/index.js]: done! I had used the PascalCase notation for the View functions, please let me know if you want me to change also the functions: onPressFacebook, onPressGoogle, onPressApple, onPressLogin, onPressRegister

File: src/translations/es.json (hunk 3 above)
Comment (dsernahiguita) [reply src/translations/es.json]: done :)

**Pull request merged**